src/vars/images.yml

```
@@ -4,7 +4,7 @@ foreman_container_image: quay.io/foreman/foreman
 foreman_container_tag: nightly
 foreman_proxy_container_image: quay.io/foreman/foreman-proxy
 foreman_proxy_container_tag: nightly
-pulp_image: quay.io/pulp/pulp-minimal:3.73
+pulp_image: quay.io/foreman/pulp:3.73
```

Do we want to adopt the Foreman tagging scheme already?
```suggestion
pulp_image: quay.io/foreman/pulp:foreman-nightly
```

Or what we really want long term:
```suggestion
pulp_image: quay.io/foreman/pulp:foreman-{{ foreman_container_tag }}
```

Now another question: why does everything have `_image` or `_tag` but Pulp basically has both combined into one?

---

> Do we want to adopt the Foreman tagging scheme already?

I'll do that following this change.